assets/sass/v2/_okta-verify.scss

```diff
@@ -24,6 +24,11 @@
   .switch-channel-link {
     color: $link-text-color;
   }
+
+  .o-form-error-container > .infobox-error + .resend-ov-link-view {
```

Comment:
can make it generic by using 
.o-form-error-container > .infobox + .resend-ov-link-view { ?

Reply:
will update